.github/workflows/main.yml

```
@@ -0,0 +1,30 @@
+name: git push into another repo to deploy to vercel
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: pandoc/latex
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install mustache (to update the date)
```

```suggestion
      - uses: actions/checkout@v4
      - name: Install mustache (to update the date)
```

최신 버전을 사용하지 않으신 이유가 있으실까요?